### ISSUE TITLE
Library dependencies installation dialog not given focus on open

### ISSUE THREAD

**Issue by per1234** [arduino/arduino-ide#1679]
### Describe the problem

Arduino libraries may specify dependencies on other libraries. When installing one of a library with defined dependencies via the Arduino IDE Library Manager, a dialog appears asking the user whether the dependencies should be installed in addition to the library:

![image](https://user-images.githubusercontent.com/8572152/202115692-76d422ee-b720-4966-8ce0-b20abe54a7df.png)

🐛 This dialog is not given focus. This makes it inaccessible for those who rely on keyboard navigation and inconvenient for those who prefer keyboard navigation.

### To reproduce

1. Open the "**Library Manager**" view.
1. Find "**ArduinoIoTCloud**" in the "**Library Manager**" view.
1. Click the <kbd>**INSTALL**</kbd> button.
1. Attempt to activate either of the buttons on the dialog using the keyboard.

🐛 The buttons are inaccessible due to the dialog not having focus.

### Expected behavior

Dialogs are always given focus on opening

### Arduino IDE version

d24a391

### Operating system

Windows

### Operating system version

10

### Additional context

Related:

- https://github.com/arduino/arduino-ide/issues/360
- https://github.com/arduino/arduino-ide/issues/1561
- https://github.com/arduino/arduino-ide/issues/1661

### Issue checklist

- [X] I searched for previous reports in [the issue tracker](https://github.com/arduino/arduino-ide/issues?q=)
- [X] I verified the problem still occurs when using the latest [nightly build](https://github.com/arduino/arduino-ide#nightly-builds)
- [X] My report contains all necessary details

**Comment by kittaakos** [comment]
Resolved via https://github.com/arduino/arduino-ide/commit/2aad0e3b163f27c51b15b58a6f35aab757a643fb.
